nasa/openmct
# fix: autoscale turned off could cause errors

Closes #5039

### Describe your changes:

ensure when autoscale is turned off there is a default range so that the UI doesn't have an error in which it is not pannable/zoomable due to undefined values. Also there were no Y axis ticks in the broken state. Now they default from -10 to 10.

- [x] Maybe a better default is to choose the current range. Yeah, I think so. I will make that change.

### All Submissions:


### Author Checklist


To test:

- make overlay plot with sine wave generator. 
- turn off autoscale for the overlay plot and save the overlay. 
- you should see ticks in the Y axis instead of no ticks (that was fixed)
- no pan the plot and panning should work (plus no error in console)

### Reviewer Checklist

* [ ] Appropriate unit tests included?
* [ ] Commit messages meet standards?
